helm/cas-bciers/templates/cronJobs/check-file-status.yaml

```diff
@@ -34,14 +34,17 @@ spec:
               - -c
               - |
                 set -euo pipefail;
-                flock -w 45 --verbose /data/locks/.check.lock \
-                .venv/bin/python manage.py check_document_file_status
+                touch /data/log/cron.log
+                (crontab -l ; echo "* * * * * flock -w 45 --verbose /data/locks/.check.lock .venv/bin/python manage.py check_document_file_status >> /data/log/cron.log 2>&1") | crontab
+                cron && timeout --preserve-status 1790s tail -F /var/log/cron.log
```

Comment:
I think a bit of documentation on what this does could be useful at some point, I've created https://github.com/bcgov/cas-reporting/issues/698